activiti-cloud-notifications-graphql-service/services/graphiql/src/main/java/org/activiti/cloud/services/notifications/graphql/graphiql/GraphiQLIndexController.java

```diff
@@ -10,9 +10,12 @@
 public class GraphiQLIndexController {
 	
 	@Value("${graphiql.index:/graphiql/graphiql.html}")
-	private String graphiqlHtml;
+    private String graphiqlHtml;
 
-    @GetMapping("/graphiql")
+    @Value("${activiti.cloud.notification.uri-prefix}")

```

Comment:
There is missing default empty value. It should be unnecessary to inject default value explicitly if there default value defined in property source for notifications service